apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/views/realm_config_view.ex

```diff
@@ -26,6 +26,12 @@ defmodule Astarte.RealmManagement.APIWeb.RealmConfigView do
     }
   end
 
+  def render("show.json", %{device_registration_limit: limit}) do
+    %{
+      data: %{device_registration_limit: limit}
```

Comment:
Shouldn't this just be `data: limit`? The auth config doesn't have an extra `%{data: %{auth_config: ...}}`